opentreemap/treemap/instance.py

```diff
@@ -573,30 +572,16 @@ def update_revs(self, *attrs):
         for attr in attrs:
             setattr(self, attr, getattr(qs[0], attr))
 
-    def itree_region_codes(self):
-        from treemap.models import ITreeRegion
+    def itree_regions(self, **extra_query):
```

Comment:
The return "type" of this method makes me a little unsure as to whether or not we should use this OO approach. Depending on the value of `itree_region_default` you are returned either a List[ITreeRegionInMemory] or an ITreeRegion QuerySet. Are there other places in the application where we duck type this intensely? Can you expand on your commit message and explain why you think this is better than constructing a list of code/name tuples?